src/workerd/api/kv.h

```diff
@@ -48,11 +48,13 @@ class KvNamespace: public jsg::Object {
   struct GetWithMetadataResult {
     GetResult value;
     kj::Maybe<jsg::Value> metadata;
+    kj::Maybe<kj::String> cacheStatus;
 
-    JSG_STRUCT(value, metadata);
+    JSG_STRUCT(value, metadata, cacheStatus);
     JSG_STRUCT_TS_OVERRIDE(KVNamespaceGetWithMetadataResult<Value, Metadata> {
       value: Value | null;
       metadata: Metadata | null;
+      cacheStatus: string | null;
```

Comment:
It became that after an optimization pass to call intern on the string rather than rely on implicit conversion, so I restored the type override. You're not going crazy :)